nss-day-cohort-43-csharp/Fullstack--span--Halen
# Fm not found page

# Description
User should see a 404 page when they go to categories and tags. 
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Please check that the 404 pages shows when you go to categories and tags.  Also make sure when you click the button it takes you to the explore page
# Checklist:
